src/applications/caregivers/components/PreSubmitInfo/index.jsx

```diff
@@ -18,7 +18,7 @@ const SignatureCheckbox = ({
       signSignature({ ...signatures, [label]: isSignatureComplete });
     },
 
-    // eslint-disable-next-line
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
cool- approving this for now. I'm not a hooks expert but recommend reading up on why this rule is in place: https://github.com/facebook/create-react-app/issues/6880#issuecomment-485912528

Reply:
Yeah I need to refactor it to use useCallback, have a ticket for it, just need to get this big boy in so my PM is happy

Reply:
My first PR kind of over flooded